cmd/helmer/main.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"context"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/kubeapps/kubeapps/cmd/helmer/internal/handler"
+	"github.com/kubeapps/kubeapps/pkg/agent"
+	"github.com/kubeapps/kubeapps/pkg/auth"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/pflag"
+	"github.com/urfave/negroni"
+	"k8s.io/helm/pkg/helm/environment"
+)
+
+var (
+	settings         environment.EnvSettings
+	chartsvcURL      string
+	userAgentComment string
+	listLimit        int
+	timeout          int64
+)
+
+func init() {
+	settings.AddFlags(pflag.CommandLine) // necessary???
+	pflag.StringVar(&chartsvcURL, "chartsvc-url", "https://kubeapps-internal-chartsvc:8080", "URL to the internal chartsvc")
+	pflag.IntVar(&listLimit, "list-max", 256, "maximum number of releases to fetch")
+	pflag.StringVar(&userAgentComment, "user-agent-comment", "", "UserAgent comment used during outbound requests") // necessary???
```

Comment:
context on why the `user-agent` is important: https://github.com/kubeapps/kubeapps/issues/767